### PR TITLE
chore(node-sdk) More docs improvements

### PR DESCRIPTION
**Pull request by roncohen**
Another round of "getting started" instructions.

### CODE REVIEW

File: packages/node-sdk/README.md [hunk 1]
```diff
@@ -70,22 +73,6 @@ if (isEnabled) {
...
-  features.huddle?.isEnabled && features.voiceHuddle?.isEnabled;
-```
-
-When using `getFeatures` be careful not to assume that a feature exists, this could be a dangerous pattern:
-
-```ts
-// warning: if the `huddle` feature does not exist because it wasn't created in Bucket
-// or because the client was unable to reach our servers for some reason, this will cause an exception:
-const { isEnabled } = boundClient.getFeatures()["huddle"];
-```
```
Comment (roncohen) [inline packages/node-sdk/README.md:88]: getting rid of this to focus on the absolutely necessary

Comment (roncohen) [reply packages/node-sdk/README.md:94]: brought this back. `getFeatures()` is useful

**Review by pavkam — APPROVED** [review]
Left a few minor nitpicks

**Comment by roncohen** [comment]
thanks!

**Pull request merged**